source/includes/fact-oplog-disk-storage.rst

```diff
@@ -4,10 +4,18 @@ sync. For example, to migrate 10 GB of data, the destination cluster must have
 at least 10 GB available for the data and another 10 GB for the insert oplog 
 entries from the initial sync.
 
-To reduce the overhead of the destination oplog entries, you can: 
+.. important:: 
+  
+   To use :ref:`embedded verification <c2c-embedded-verifier>`, you must have a 
+   larger oplog on the destination. If you enable the embedded verifier and 
+   reduce the size of the destination oplog, the embedded verifier can fall off 
+   the oplog.
+
+If you need to reduce the overhead of the destination oplog entries and the 
```

Comment:
Consider rewriting to avoid "overhead".

Reply:
I'm leaning towards leaving this as is because I feel that "overhead" might be the best word to encapsulate both "size" and "retention period" of the oplog. Also we've used the word a large number of times across our docs.